draft-ietf-taps-interface.md

```diff
@@ -1612,19 +1612,19 @@ Many Connection Properties have a corresponding Selection Property that
 enables applications to express their preference for protocols providing a supporting transport feature.
 
 
-### Retransmission Threshold Before Excessive Retransmission Notification {#conn-excss-retransmit}
+### Loss Threshold before Excessive Loss Notification {#conn-excss-loss}
 
 Name:
-: retransmitNotifyThreshold
+: excessLossNotifyThreshold
 
 Type:
-: Integer, with special value `Disabled`
+: Numeric, with special value `Disabled`
 
 Default:
 : Disabled
 
-This property specifies the number of retransmissions required to cause the application
-to be notified of `Excessive Retransmissions`.
+This property specifies the number of lost packtes per second required to cause the application
+to be notified of `Excessive Loss`.
```

Comment:
```suggestion
to be notified of `Excessive Retransmission`. In case of unreliable data transfer, configuring this threshold will not have any effect.
```